packages/common/src/utils/mathUtils.ts

```diff
@@ -68,3 +68,7 @@ export const rotate = (rot: Quaternion, x: number, y: number, z: number) => {
 export const eulerToQuaternion = (x, y, z, order = 'XYZ') => {
   return new Quaternion().setFromEuler(new Euler(x, y, z, order))
 }
+
+export const isZero = (v: Vector3): boolean => {
```

Comment:
replace with Vector3.isEqual against an empty vector3

Reply:
@HexaField wasn't this better the way it was before? now a new vector3 is being made every time this is called.